_test/test_change_crystal/test_change_crystal_1.m

```diff
@@ -9,9 +9,9 @@
 
 % -----------------------------------------------------------------------------
 % Add common functions folder to path, and get location of common data
-common_path = add_common_test_functions_to_path();
-cleanup = onCleanup(@() rmpath(common_path));
-common_data_dir=fullfile(horace_root(),'_test','common_data');
+HORACE_ROOT = horace_root();
+addpath(fullfile(HORACE_ROOT,'_test','common_functions'))

```

Comment:
I've reverted this file back to its original state. 

I thought we'd agreed to tackle these instances by adding a `hor_config.init_tests` setting, so that we could add the `common_functions` directory to the path when tests were enabled? That would remove the need for these `addpath` calls completely. I'll raise a ticket if that plan's OK with you?

Reply:
yup, not sure where this comment came from... maybe a draft one.